.nuget/NuGet.targets

```diff
@@ -7,7 +7,7 @@
         <RestorePackages Condition="  '$(RestorePackages)' == '' ">false</RestorePackages>
 
         <!-- Property that enables building a package from a project -->
-        <BuildPackage Condition=" '$(BuildPackage)' == '' ">false</BuildPackage>
+        <BuildPackage Condition=" '$(BuildPackage)' == '' ">true</BuildPackage>
```

Comment:
Tried to run nuget manually and it doesn't like `.csproj` with `.teamcity` extension.
I enabled nuget build here. So it probably should be disabled for teamcity.